h/auth/__init__.py

```diff
@@ -41,6 +44,15 @@ def includeme(config):
     global DEFAULT_POLICY
     global WEBSOCKET_POLICY
 
+    config.register_service_factory('.services.auth_ticket_service_factory',
+                                    iface=pyramid_authsanity.interfaces.IAuthService)
+
+    # Set up authsanity
+    settings = config.registry.settings
+    settings['authsanity.source'] = 'cookie'
+    settings['authsanity.secret'] = derive_key(settings['secret_key'], b'h.auth')
```

Comment:
We can probably be a bit more specific than `b'h.auth'`. Let's call it something like `b'h.auth.cookie_secret'`.